### ISSUE TITLE
On Android (not UWP) player stops after a few times

### ISSUE THREAD

**Issue by odahan** [adrianstevens/Xamarin-Plugins#6]
After playing a bit, player stops to emit sound on Android (high version since there's a bug for olders, I already open a bug for this specific problem).
After searching a bit to help you, it seems that under undroid a player must reset to be able to play again. The player interface you create does not have a method for this purpose.
The best is certainly to expose the method with an Android implementation and an empty one for UWP (that works ok on UWP).
From Stackoverflow "...When you are done with it call mp.release() so that it can release the resources..."
So adding a "Release()" method that user must call when loading a new stream seems a quite good and simple solution.
Post reference : https://stackoverflow.com/questions/18773515/mediaplayer-stops-working-after-30-uses
Thanks again !

**Comment by adrianstevens** [comment]
thank you @odahan - I'll get this added into the next release

**Comment by adrianstevens** [comment]
1st - I was able to reproduce the issue
2nd - thank you for looking into this and sending the stack overflow link

I took a closer look at the Android MediaPlayer state diagram https://developer.android.com/reference/android/media/MediaPlayer.html and it appears we should be calling 
~~~
player.SeekTo(0);
player.Stop();
player.Prepare();
~~~
after playback has completed. I've been testing locally and that seems to fix the audio playback issue at the expense of a bit of latency - which should only be noticeable if you're playing sounds rapidly (I've been testing with the DrumPad code).

Let's try this fix but I'll leave the issue open for now.

I will also add the Release method (or something similar). I think it'll be useful if you want to change the audio source on the same MediaPlayer instance.
